### ISSUE TITLE
setup fail2ban

### ISSUE THREAD

**Issue by ezraholm50** [ezraholm50/BerryCloud#12]

**Comment by enoch85** [comment]
What do you mean? Check which timezone the user has and then add it to config.php in onwcloud?

**Comment by enoch85** [comment]
Here is something to look into: http://www.cyberciti.biz/faq/linux-unix-set-tz-environment-variable/

**Comment by ezraholm50** [comment]
Yes exactly. I set the reconfigure tzdata at the top of the setup.sh script so the correct timezone gets picked by the user.

Now the only thing we need to figure out is the same as with take for example $ADDRESS, stripe all the text we don't need and only use the time zone.

If you run ¨data" you'll get enough info. I'll investigate...

**Comment by enoch85** [comment]
**PRO TIP**: Test each part of the script individually.

**Comment by ezraholm50** [comment]
Done already, its a distro/repo issue

**Comment by ezraholm50** [comment]
temp fix, install from downloaded package. 
Needs to be figured out what the repo issue is.
